rskj-core/src/main/java/org/ethereum/rpc/CallArguments.java

```diff
@@ -36,6 +38,7 @@ public class CallArguments {
     private String nonce;
     private String chainId;
     private String type; // ignore, see https://github.com/rsksmart/rskj/pull/1601
+    private String input;
```

Comment:
do we really need this extra field? Can we reuse `data` and have something like this?:
```
    public void setInput(String input) {
        this.data = input;
    }
```

Reply:
True, I've removed it and implemented your suggested approach.